teuthology/orchestra/remote.py

```diff
@@ -206,6 +206,23 @@ def run(self, **kwargs):
         r.remote = self
         return r
 
+    def mkdtemp(self, suffix=None, parentdir=None):
+        """
+        Create a temporary directory on remote machine and return it's path.
+        """
+        args = ['mktemp', '-d']
+
+        if suffix is not None:
+            args.append('--suffix')
+            args.append(suffix)
+
+        if parentdir is not None:
+            args.append('-p')
+            args.append(parentdir)
+
+        proc = self.run(args=args, stdout=StringIO())
```

Comment:
Could you please use `self.sh` method it specifically exists for such kind of tiny code snippets.

Reply:
Done.